app/assess/templates/sub_criteria.html

```diff
@@ -1,105 +1,70 @@
-
-                    <h2 class="govuk-heading govuk-heading-l" id="comments-heading">Comments</h2>
-                    <section class="comment-section">
-                        <h3 class="govuk-heading-m">Community engagement</h3>
-
-                        <p class="govuk-body-s">There are currently no comments&nbsp;for this section.
-                            {# <a href="#" class="govuk-link govuk-link--no-visited-state">Add a comment</a> #}
-                        </p>
-
-                    </section>
-                    <section class="comment-section">
-                        <h3 class="govuk-heading-m">Local support</h3>
-
-                        <p class="govuk-body-s">There are currently no comments&nbsp;for this section.
-                            {# <a href="#" class="govuk-link govuk-link--no-visited-state">Add a comment</a> #}
-                        </p>
-
-                    </section>
-
-
+                <div class="govuk-grid-column-full">
+                    <div class="govuk-grid-column-two-thirds">
+                        <h1 class="govuk-heading-xl fsd-banner-content">Fund: {{ fund }}</h1>
+                        <h2 class="govuk-heading-l fsd-banner-content">Project reference: {{ project_reference }}</h2>
+                        <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">Project name: {{ project_name }}</h3>
+                        <h3 class="govuk-body-l fsd-banner-content">Total funding requested: &pound;{{ "{:,.2f}".format(funding_amount_requested) }}</h3>
+                        <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">{{ workflow_status | status_to_human }}</h3>
+                    </div>
                 </div>
-
             </div>
-
         </div>
-
+    </div>
+        <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third content-container">
+            <h1 class="govuk-heading-l scoring-heading">{{sub_criteria.id}}</h1>
+        </div>
     </div>
 
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <div class="govuk-grid-column-one-third">
+                {{ navbar(sub_criteria,themes, current_theme_id, on_summary) }}
+            </div>
+            <div class="theme govuk-grid-column-two-thirds">
+                <h2 class="govuk-heading-m">{{current_theme_id|capitalize}}</h2>
+                <h3 class="govuk-heading-s response-title">Applicant's response</h3>
+                <div class="template-answer">
+                    <h2 class="govuk-heading-s"><strong>Question One</strong></h2>
+                    <p class="govuk-body">Answer One</p>
+                    <h2 class="govuk-heading-s"><strong>Question Two</strong></h2>
+                    <p class="govuk-body">Answer Two</p>
+                </div>
+                <div class="govuk-!-margin-top-7" id="anchor">
```

Comment:
Comments part here are my changes, the rest are from https://github.com/communitiesuk/funding-service-design-assessment/pull/62 for the base template